src/PrestaShopBundle/Translation/Factory/ThemeTranslationsFactory.php

```diff
@@ -80,7 +80,7 @@ public function createTranslationsArray($themeName, $locale = 'en_US')
                 ;
 
                 $translations[$domain][$translationKey] = array(
-                    'xlf' => $translations[$domain][$translationKey],
+                    'xlf' => $translationKey != $translationValue ? $translations[$domain][$translationKey] : '',
```

Comment:
Yes, we need the default catalogue in case we have set custom translations and we want to reset to the default ones.